test/app/renderer/api/newMnemonics.spec.ts

```diff
@@ -0,0 +1,45 @@
+import * as api from "app/renderer/api"
+import {ipcRendererFactory} from "test/__stubs__/ipcRenderer"
+import {jsonSerializer} from "test/__stubs__/serializers"
+import {newMnemonics} from "app/renderer/api/newMnemonics"
+import {routes} from "app/main/api"
+
+describe("NewMnemonics API", () => {
+  const mnemonics = {
+    mnemonics: "narrow flavor sense humble radar rail rail cactus radar broom oyster gym"
+  }
+  const messageHandler = jest.fn()
+  messageHandler.mockResolvedValue(mnemonics)
+  const options = {
+    ...api.defaultOptions,
+    timeout: 0,
+    idGen: () => "some generated id",
+    json: jsonSerializer,
+    ipc: ipcRendererFactory(),
+    messageHandler
+  }
+  const client = new api.Client(options)
+
+  it("should return valid mnemonics", async () => {
+    const response = await newMnemonics(client)
+    expect(response).toMatchObject(mnemonics)
+  })
+
+  it("should send a correct routed JSON-RPC request", async () => {
+    const expected = {
+      jsonrpc: "2.0",
+      id: "some generated id",
+      method: "newMnemonics",
+      params: []
+    }
+
+    await newMnemonics(client)
```

Comment:
This line could use a comment because:
- It's a statement
- It's asynchronous
- Returned value is discarded
- People tend to read tests as library usage examples